stepper/src/main/java/com/aceinteract/android/stepper/StepperNavigationView.kt

```diff
@@ -224,6 +224,22 @@ class StepperNavigationView(context: Context, attrs: AttributeSet) : FrameLayout
         addView(menu)
     }
 
+    /**
+     * Sets up the stepper to navigate steps without using navigation components.
+     */
+    fun setupWithoutNavController() {
```

Comment:
There is no need for this, just move this to the original initialization for `onStepChanged` on line 126 and replace the when with if/else

Like this:
```kotlin
onStepChanged = {
    if (it == menu.size()) {
        stepperNavListener?.onCompleted()
    } else {
        menu.currentStep = it
        menu.updateUI()
        stepperNavListener?.onStepChanged(currentStep)
    }
}
```

Reply:
You are absolutely right.